.github/workflows/assign-issues.yml

```
@@ -1,23 +1,36 @@
     runs-on: ubuntu-latest
-    permissions: 
+    permissions:
       issues: write
       pull-requests: write
     steps:
-      - name: OWASP BLT Action
-        uses: OWASP-BLT/BLT-Action@main
+      - uses: actions/checkout@v4
+      
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)